Makefile

```diff
@@ -52,22 +52,24 @@ lint: $(SRC)
 vet: $(SRC)
 	@go vet ./...
 
-minisetup: miniup miniimage helm
+k3s_mac_lima_create: 
+	limactl start --tty=false ./resources/lima/k3s.yaml
 
-miniup: ## start minikube
-	@minikube start --kubernetes-version=$(K8S_VERSION) --cpus 2 --memory 4096
+k3s_mac_lima_start:
+	limactl start k3s
 
-minidown: ## stop minikube
-	@minikube stop
+k3s_mac_lima_helm:
+	mkdir -p "$${HOME}/.lima/k3s/conf"
+	limactl shell k3s sudo cat /etc/rancher/k3s/k3s.yaml >$${HOME}/.lima/k3s/conf/kubeconfig.yaml
+	@helm upgrade --install --namespace operator --create-namespace my-dboperator charts/db-operator -f charts/db-operator/values.yaml -f charts/db-operator/values-local.yaml --kubeconfig $${HOME}/.lima/k3s/conf/kubeconfig.yaml
+	echo "Don't forget to use k3s docker coonfig \nexport KUBECONFIG=$${HOME}/.lima/k3s/conf/kubeconfig.yaml"
 
-minidelete: ## delete minikube
-	@minikube delete
+k3s_mac_deploy: build k3s_mac_image k3s_mac_lima_helm
 
-minidashboard: ## open minikube dashboard
-	@minikube dashboard
-
-miniimage: build
-	@minikube image load my-image.tar
+k3s_mac_image:
+	limactl copy my-image.tar k3s:/tmp/db.tar
+	limactl shell k3s sudo k3s ctr images import /tmp/db.tar
+	limactl shell k3s rm -f /tmp/db.tar
```

Comment:
what about adding `deploy`, `image`, `setup` ... and make it different depends on system? for example
```
image: 
	ifeq ($(UNAME_S),Darwin)
		k3s_mac_image
        else
                 k3d_image
        endif
```

Reply:
I wouldn't do that unless everybody uses k3s with lima. It doesn't seem like a common solution for now. And after this [PR](https://github.com/k3d-io/k3d/issues/593) is merged, we'll be able to k3s on MacOS as well.